src/terminal/terminal/terminal-priv.h

```diff
@@ -269,6 +269,11 @@ struct guac_terminal {
      */
     int visible_cursor_col;
 
+    /**
+     * Backup of cursor position when using alternate buffer.
+     */
+    int cursor_row_alt, cursor_col_alt, visible_cursor_row_alt, visible_cursor_col_alt;
+
```

Comment:
As the matching primary members are declared individually, I think this should probably follow the same convention and declare each of these individually.